toast-tk-runtime/src/main/java/io/toast/tk/runtime/block/TestBlockRunner.java

```diff
@@ -177,12 +229,20 @@ private String updateCommandWithVarValues(final ActionAdaptaterLocator actionAda
 		String outCommand = actionAdaptaterLocator.getTestLineDescriptor().getActionImpl();
 		final int groupCount = matcher.groupCount();
 		final Object[] args = new Object[groupCount];
+		final int maxSize = 10000;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)